packages/elements/src/components/ino-markdown-editor/ino-markdown-editor.scss

```diff
@@ -149,6 +153,15 @@ ino-markdown-editor {
     border-radius: 4px;
     border: 1px solid theme.$n-2;
     color: rgba(0, 0, 0, 0.87);
+    font-size: var(--ino-markdown-editor-paragraph-font-size, 1rem);
+
+    h1 {
+      font-size: var(--ino-markdown-editor-h1-font-size, 2rem);
+    }
+
+    h2 {
+      font-size: var(--ino-markdown-editor-h2-font-size, 1.75rem);
+    }
```

Comment:
I've got an idea that would make it a little more consumer-friendly: Instead of assigning each text type (h1, h2, p) its own value, how about offering a variable that calculates the other values based on it? This has the advantage that I only have to enter one value as a consumer (e.g. `12px`) and the other values can be calculated automatically (e.g. `h1 = 12px * 1.5`, `h2 = 12px * 1.25`, ...). Otherwise, as a consumer, I always have to specify these 3 variables at the same time, as the ratio between the font sizes is would no longer be correct.

Reply:
Good idea - I agree, by dynamically scaling with `calc()`, the user only needs to specify one base font size without much thought, while maintaining the correct relative ratios of `h1` and `h2`.